site/_posts/2022-07-18-manageiq-najdorf-ga-announcement.md

```diff
@@ -0,0 +1,73 @@
+---
+title: ManageIQ Najdorf GA Announcement
+author: Fryguy
+date: 2022-07-18
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, I'm happy to announce the release of **ManageIQ Najdorf**!  This release is named after the chess grandmaster [Miguel Najdorf](https://en.wikipedia.org/wiki/Miguel_Najdorf).  You can download the Najdorf-1 release on our [downloads page](/download).
+
+Here are the highlights of the [Najdorf release](https://github.com/orgs/ManageIQ/projects/13#column-12963689):
+
+* New Providers
+  * IBM Power HMC provider [[#21475]](https://github.com/ManageIQ/manageiq/pull/21475)
+  * IBM Cloud Infrastructure Center (CIC) provider [[#21631]](https://github.com/ManageIQ/manageiq/pull/21631)
+  * Cisco Intersight Provider [[#21715]](https://github.com/ManageIQ/manageiq/issues/21715)
+  * VMware Tanzu Kubernetes Grid (TKG) [[#703]](https://github.com/ManageIQ/manageiq-providers-vmware/issues/703)
+* Provider Enhancements
+  * IBM Cloud [[#263]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/issues/263)
+    * Expanded network inventory collection (public gateways, VPN gateways, load balancers, ACLs, security group rules)
+    * Support to modify/resize VM [[#300]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/300)
+    * Metrics collection [[#308]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/308)
+    * Catalog service provisioning [[#362]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/362)
+    * Create/Delete cloud databases [[#364]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/364), [[#366]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/366)
+    * Add metrics collection support for IKS [[#314]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/issues/314)
+  * IBM Power HMC [[#8]](https://github.com/ManageIQ/manageiq-providers-ibm_power_hmc/issues/8)
```

Comment:
I'm not sure I should include this sections at all, since this is a new provider as per the New Providers section.

Reply:
Maybe we should move these individual features into the section above then?  Just the inclusion on the provider doesn't tell the whole story

Reply:
I thin for now I'm going to leave this.  I have some other ideas around getting things on the Roadmap more specifically that I'd like to discuss with you and I think this might fall into that bucket.